packages/react-meteor-data/useFind.ts

```diff
@@ -59,47 +59,40 @@ const checkCursor = <T>(cursor: Mongo.Cursor<T> | Partial<{ _mongo: any, _cursor
   }
 }
 
-const useFindClient = <T = any>(factory: () => (Mongo.Cursor<T> | undefined | null), deps: DependencyList = []) => {
-  let [data, dispatch] = useReducer<Reducer<T[], useFindActions<T>>>(
-    useFindReducer,
-    []
-  )
-
-  const { current: refs } = useRef<{ useReducerData: Boolean, data: T[] }>({ useReducerData: false, data: [] })
+const useFindClient = <T = any>(
+  factory: () => Mongo.Cursor<T> | undefined | null,
+  deps: DependencyList = []
+) => {
+  const cursor = useMemo(() => {
+    const cursor = Tracker.nonreactive(factory);
+    if (Meteor.isDevelopment) {
+      checkCursor(cursor)
+    }
+    return cursor
+  }, deps)
 
-  const cursor = useMemo(() => (
-    // To avoid creating side effects in render, opt out
-    // of Tracker integration altogether.
-    Tracker.nonreactive(() => {
-      refs.useReducerData = false
-      const c = factory()
-      if (Meteor.isDevelopment) {
-        checkCursor(c)
+  const [data, dispatch] = useReducer<Reducer<T[], useFindActions<T>>, null>(
+    useFindReducer,
+    null,
+    () => {
+      const data: T[] = []
+      if (cursor instanceof Mongo.Cursor) {
+        const observer = cursor.observe({
+          addedAt(document, atIndex, before) {
```

Comment:
All of the functions have space after their name in the `observe` call below - let's keep them in align.